modules/stochastic_tools/doc/content/modules/stochastic_tools/examples/bayesian_uq_1d_diff.md

```diff
@@ -0,0 +1,207 @@
+
+## Inferring Calibration Parameters Only (Fixed $\sigma$ Term)
+
+### Sub-Application Input
+
+The problem defined above, with respect to the [MultiApps](MultiApps/index.md) system, is a sub-application. The complete input file for the problem is provided in [mcmc-sub]. The only item required
+to enable the stochastic analysis is the [Controls](Controls/index.md) block, which contains a
+[SamplerReceiver](/SamplerReceiver.md) object, the use of which will be explained in the following section.
+
+!listing modules/stochastic_tools/test/tests/samplers/mcmc/sub.i
+         id=mcmc-sub
+         caption=Complete input file for executing the transient diffusion problem.
+
+
+### Master Input
+
+The master application, with respect to the [MultiApps](MultiApps/index.md) system, is the driver of the stochastic simulations. The master input does not perform a solve by itself, but has to be coupled with the sub-application for a full stochastic analysis. The complete input file for the master application using the base MCMC class is shown in [mcmc-base-master]. Details of each indivisual input section will is explained in details as follows.
+
+1. The [StochasticTools](StochasticTools/index.md) block is defined at the beginning of the master application to automatically create the necessary objects for running a stochastic analysis. 
+
+!listing modules/stochastic_tools/test/tests/samplers/mcmc/main_base.i block=StochasticTools
+
+2. The [Distributions](Distributions/index.md) block is used to define the prior distributions for the two stochastic boundary conditions. In this example, the two stochastic boundary conditions are assumed to follow normal distributions 
+
+!listing modules/stochastic_tools/test/tests/samplers/mcmc/main_base.i block=Distributions
+
+3. A [Likelihoods](Likelihood/index.md) object is created to specify the type of likelihood function used in the analysis. 
+
+- A Gaussian-type log-likelihood function is used in the current analysis, as specified by `type = Gaussian` and `log_likelihood = true`. 
+- The `noise` term defines the experimental noise plus model deviations against experiments. Variance of the Gaussian distribution is fixed to a constant value of 0.05 through the `noise_specified` term, as will be explained next in the [Reporters](Reporters/index.md) block. Note that this term can be inferred 
+- The `file_name` argument takes in a CSV file that contains experimental values. In this example, the `exp_0.05.csv` file contains 8 independent experimental measurements, which are the sum of pre-simulated solution and a Gaussian noise with standard deviation of 0.05. 
+
+!listing modules/stochastic_tools/test/tests/samplers/mcmc/main_base.i block=Likelihood
+
+4. In the [Samplers](Samplers/index.md) block, [PMCMCBase](PMCMCBase.md) is used to sample the left and right boundary conditions according to the parallelized Markov chain Monte Carlo method at certain configuration conditions. 
```

Comment:
"PMCMCBase" does not perform MCMC sampling. The sampler should be one of IGMH, AffineInvariantSS, or AffineInvariantDES. Please change the .i file correspondingly. You may pick one among the three, and preferably DES.